other/text_to_speech.py

```diff
@@ -0,0 +1,137 @@
+"""
+Created by: Ariel Sperduti
+
+Given a text it converts into an  MP3 audio file.
+Under the hood it uses fromtexttospeech.com.
+
+Example:
+    text_to_speech(text="¡Hola Mundo!", language="Spanish", voice="Mateo")
+"""
+
+import re
+from typing import Optional
+
+import requests
+
+LANGUAGES_VOICES = {
+    "US English": ["Alice", "Daisy", "George", "Jenna", "John"],
```

Comment:
```suggestion
    "US English": ("Alice", "Daisy", "George", "Jenna", "John"),
```
Tuples take less RAM than lists and they indicate to the reader that the values will not change at runtime.